_layouts/default.html

```diff
@@ -8,7 +8,13 @@
 
     <link rel="icon" type="image/x-icon" href="/assets/images/favicon.ico">
     <link rel="stylesheet" href="/assets/main.css">
-    <script src="/assets/js/application.js"></script>
+    {% if page.script %}
+      <script src="/assets/js/{{ page.script }}"></script>
```

Comment:
can we do it in one line?
maybe: `page.script || 'application.js'`

Reply:
I tried, it didn't work, I will try again